src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesListItem.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  Flex,
+  FlexItem,
+  Label,
+  LabelGroup,
+} from '@patternfly/react-core';
+import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon';
+
+import { useTranslation } from '@rhoas/app-services-ui-components';
+
+export type ConnectorTypesListItemProps = {
+  id: string;
+  labels: string[];
+  title: string;
+  verison: string;
+  numLabels: number;
+};
+
+export const ConnectorTypeListItem: React.FunctionComponent<ConnectorTypesListItemProps> =
+  ({ id, labels = [], title, verison, numLabels }) => {
+    const { t } = useTranslation();
+
+    return (
+      <DataList aria-label="selectable data list connector">
+        <DataListItem aria-labelledby={id} id={id}>
+          <DataListItemRow>
+            <Flex justifyContent={{ default: 'justifyContentFlexStart' }}>
+              <FlexItem>
+                {labels.includes('source') ? (
+                  <i
+                    style={{
+                      fontSize: 'var(--pf-global--icon--FontSize--lg)',
+                      color: 'var(--pf-global--Color--200)',
+                    }}
+                    className={'pf-icon pf-icon-data-source'}
+                  ></i>
+                ) : (
+                  <i
+                    style={{
+                      fontSize: 'var(--pf-global--icon--FontSize--lg)',
+                      color: 'var(--pf-global--Color--200)',
+                    }}
+                    className={'pf-icon pf-icon-data-sink'}
+                  ></i>
+                )}
+              </FlexItem>
+              <FlexItem>
+                <DataListItemCells
+                  dataListCells={[
+                    <DataListCell key="primary content">
+                      <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                          <Flex>
+                            <FlexItem>
+                              <strong
+                                className="pf-u-font-weight-bold"
+                                id="selectable-action-item1"
+                              >
+                                {title}
+                              </strong>
+                            </FlexItem>
+                            <FlexItem>
+                              <OutlinedQuestionCircleIcon
```

Comment:
Here is where the description field is used, think [this can](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGalleryCard.tsx#L104-L111) be used as-is